quilt/tools/core.py

```diff
@@ -2,8 +2,25 @@
 import hashlib
 import struct
 
+# for check functions
+import pandas
```

Comment:
`core.py` is actually shared with the backend (by copying the file... yeah, we need to fix that). So don't add client-specific code here.

Reply:
COMMENTED - added a comment about this.  But I also left the code because the checks will eventually need to run server-side (in a container...)